src/lib/components/chat-message.svelte

```diff
@@ -3,6 +3,7 @@
 	export let myMessage = false
 	export let object = false
 	export let group = false
+	export let sameSender = false
 	export let sender: undefined | string = undefined
```

Comment:
I am starting to feel like there is a lot of flags. Can you add comment to them what they indicate? I am starting to get lost.